cobalt/devinfra/kokoro/bin/cleanup.sh

```diff
@@ -1,12 +1,9 @@
 set -x
 
-REPO_ROOT="$(dirname "$0")/../../.."
 # After kokoro build finishes, changed files in c:/tmpfs get rsync'ed to a borg executor,
 # from where the artifacts and reports are processed.
 # Especially on Windows, the rsync can take long time, so we cleanup the cobalt workspace
 # after finishing each build.
 set +e
-pushd $REPO_ROOT
-time git clean -fdx
-popd
+time git -C "${WORKSPACE_COBALT}" clean -dfx
```

Comment:
Is `$WORKSPACE_COBALT` set here? It's not exported from `build.sh` where it's defined.

Reply:
Yes, it's exported in the build scripts:
https://github.com/youtube/cobalt/blob/3b068ece/cobalt/devinfra/kokoro/bin/dind_builder_runner.sh#L29
https://github.com/youtube/cobalt/blob/3b068ece/cobalt/devinfra/kokoro/bin/dind_runner.sh#L29